includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -17,14 +17,20 @@ public function getData() {
 		global $wgUser;
 
 		$data = [
+			// TODO: restore old logo before 4th October 2016
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1135)